pkg/csi/service/wcp/controller.go

```diff
@@ -245,6 +271,57 @@ func (c *controller) ReloadConfiguration() error {
 	return nil
 }
 
+// ReconnectVC re-establishes VC connection
+func (c *controller) ReconnectVC() error {
```

Comment:
What's the delta between this function and `ReloadConfiguration`? Can't the latter be reused on line 160?

Reply:
I refactored the current ReloadConfiguration function to be re-used for ca file rotation